plugins/servicenow/icon_servicenow/actions/get_incident_attachment/action.py

```diff
@@ -25,4 +26,4 @@ def run(self, params={}):
 
         response = self.connection.request.make_request(url, method)
 
-        return {Output.ATTACHMENT_CONTENTS: response.get("resource")}
+        return {Output.ATTACHMENT_CONTENTS: str(base64.b64encode(response.get("resource")), "utf-8")}
```

Comment:
This is going to raise an exception if the status code is a `204` and the `make_request` function returns `resource` as `None`.